File: Beginner/The Block Game (PALL01)/block.java
import java.util.*;

class Codechef {
	public static void main (String[] args) throws java.lang.Exception {

		Scanner scan = new Scanner (System.in);
		if(scan.hasNext()){
		    int t = scan.nextInt();
		    for(int i=0; i<t; i++){
		        int num = scan.nextInt();
		        int rev=0, d=0;
		        int temp = num;
		        
		        while(temp!=0){
		            d = temp%10;
		            temp /= 10;
		            rev = rev*10+d;
		        }
		        if(rev==num){
		            System.out.println("wins");
		        }
		        else{
		            System.out.println("loses");
		        }
		    }
        }
        scan.close();
	}
}

File: Beginner/The Block Game (PALL01)/block.cpp
#include <iostream>
using namespace std;

int main() {
	int t;
	cin>>t;
	while(t--) {

	    int num, rev=0, temp;
	    cin >> num;
	    temp = num;
	    
	    while(temp>0) {
            rev = rev*10 +temp %10;
            temp = temp/10;
	    }
	    
	    if(rev==num)
	        cout<<"wins"<<endl;
	    else
	        cout<<"loses"<<endl;
	}
	return 0;
}